packages/accounts/src/account.ts

```diff
@@ -291,10 +290,10 @@ export class Account {
                 finality: 'optimistic'
             });
 
-            // store nonce as BN to preserve precision on big number
+            // store nonce as BigInt to preserve precision on big number
             const accessKey = {
                 ...rawAccessKey,
-                nonce: new BN(rawAccessKey.nonce),
+                nonce: BigInt(rawAccessKey.nonce || 0)
```

Comment:
is this the intended behaviour? this will default to 0 if `nonce` is not defined, was that the case for `new BN(undefined)`? 

This is very relevant, specially since it is inside a `try/catch` statement

Reply:
I tested it myself, and indeed, `new BN(undefined) === BN(0)`

Reply:
will leave it `nonce: BigInt(rawAccessKey.nonce || 0)` because `BigInt` constructor doesn't work with `undefined`